src/python/pants/bin/local_pants_runner.py

```diff
@@ -238,22 +242,15 @@ def _print_help(self, request: HelpRequest) -> ExitCode:
         return help_printer.print_help()
 
     def _get_workunits_callbacks(self) -> Tuple[WorkunitsCallback, ...]:
-        # Load WorkunitsCallbacks with the legacy `get_streaming_workunit_callbacks` method, and with
-        # the new WorkunitsCallbackFactories method.
-        # NB: When the `--streaming-workunits-handlers` deprecation triggers, the first method will be
-        # removed.
-        streaming_handlers = self.options.for_global_scope().streaming_workunits_handlers
-        callbacks = list(Subsystem.get_streaming_workunit_callbacks(streaming_handlers))
+        # Load WorkunitsCallbacks by requesting WorkunitsCallbackFactories, and then constructing
+        # a per-run instance of each WorkunitsCallback.
         (workunits_callback_factories,) = self.graph_session.scheduler_session.product_request(
```

Comment:
is this covered in tests? 
can we assert (under test) that if there is a `WorkunitsCallback` added it will get loaded?

Reply:
Good call. Added in the third commit.